betterdgg/modules/spooky.js

```diff
@@ -8,12 +8,14 @@
         "INFESTINY": [ 'crawl' ],
         "NoTears": [ 'pulse' ],
         "SURPRISE": [ 'fadeIn', 'pulse', 'spin' ],
-        "WhoahDude": [ 'blink' ]
+        "WhoahDude": [ 'blink' ],
+        "YEE": [ 'crawl' ],
+        "NOBULLY": [ 'blink' ] //TODO change effect
     };
     var EMOTE_RE = new RegExp("\\b(?:bdgg-)?chat-emote-(" + Object.keys(EMOTES).join('|') + ")");
 
     bdgg.spooky = (function() {
-        var END = moment('2014-11-01 05:00');
+        var END = moment('2016-11-01 05:00'); //TODO verify time and possibly take care of timezones
```

Comment:
Its already done here, https://github.com/PurpleCox/betterdgg/blob/highcolour/betterdgg/modules/spooky.js 
 just other parts did not work so I told @xDashh and @BryceMatthes to fix that first 👌